spec/kaiserfile_spec.rb

```diff
@@ -170,4 +170,39 @@
       end
     end
   end
+
+  context '#service' do
+    context 'when service is specified' do
+      let(:kaiserfile) { "service 'santaclaus'" }
+
+      it 'adds a service' do
+        kaiserfile = Kaiser::Kaiserfile.new('Kaiserfile')
```

Comment:
No action needed for this PR since the spec was already like this, but I was confused at first glance since it looks like `kaiserfile` is being declared then immediately overwritten :sweat_smile:.

Reply:
o yeah this is so strange